frontend/src/components/eceweApplication/EceweEligibility.vue

```diff
@@ -377,9 +377,20 @@ export default {
     ...mapMutations('navBar', ['forceNavBarRefresh']),
     isReadOnly(question) {
       if (this.isChangeRequest) {
-        if (question == 'optInECEWE') {
+        let currentCR = this.userProfileChangeRequests?.filter(el=>el.changeRequestId===this.changeRequestId)[0];
+        if(currentCR.unlockEcewe){
```

Comment:
should we have null check here? just in case, in some odd event, currentCR is null.